src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml

```diff
@@ -61,84 +111,52 @@
                 </Button>
                 <ComboBox ItemsSource="{Binding Source={StaticResource engineEnum}, Mode=OneWay}"
                           SelectedItem="{Binding nodeModel.Engine, Mode=TwoWay, RelativeSource={RelativeSource AncestorType=dww:ModelessChildWindow}}"
-                          Style="{StaticResource ResourceKey=SComboBox}"
-                          Name="EngineSelectorComboBox"
                           SelectionChanged="OnEngineChanged"
-                          VerticalAlignment="Bottom"
-                          VerticalContentAlignment="Center"
-                          HorizontalContentAlignment="Left"
-                          FontSize="14px"
-                          Height="40"
-                          Width="Auto"
-                          Margin="1"
-                          Foreground="{StaticResource DynamoStandardLableTextBrush}"
-                          Visibility="Collapsed" 
-                          ToolTip="{x:Static p:Resources.PythonScriptEditorEngineDropdownTooltip}"/>
-                <Button Style="{DynamicResource ResourceKey=SButtonWithShapeIcon}"
+                          Style="{StaticResource DropDown}"
+                          Name="EngineSelectorComboBox"
+                          Visibility="Collapsed" />
+                <Button Style="{StaticResource IconButton}"
                         Name="MigrationAssistantBtn"
                         Click="OnMigrationAssistantClicked"
-                        VerticalAlignment="Bottom"
-                        Margin="1"
                         ToolTip="{x:Static p:Resources.PythonScriptEditorMigrationAssistantButtonTooltip}">
                     <Button.Resources>
                         <Image x:Key="Shape"
-                               Width="40"
+                               Width="44"
                                Source="/PythonNodeModelsWpf;component/Resources/2to3Icon.png" />
                     </Button.Resources>
                 </Button>
             </StackPanel>
             <StackPanel Orientation="Horizontal"
                         HorizontalAlignment="Right"
                         Grid.Column="1">
-                <Button Content="{x:Static p:Resources.PythonScriptEditorMoreInfoButton}"
```

Comment:
Can you remove the resource string as well? If they are only used at button content, I think they are